.github/workflows/build-revive-wasm.yml

```diff
@@ -118,16 +117,9 @@ jobs:
         run: |
           Set-ExecutionPolicy RemoteSigned -Scope CurrentUser
           iex (new-object net.webclient).downloadstring('https://get.scoop.sh')
-          scoop install bun@${{ env.BUN_VERSION }}
```

Comment:
Name of the job still correct?

Reply:
Good catch thanks! I probably should have removed the whole thing instead.

Reply:
I think that you need to keep wget

Reply:
Ah so it wasn't used only for downloading bun?